mgmt_cli_test.py

```diff
@@ -87,6 +87,21 @@ def _create_keyspace_and_basic_table(self, keyspace_name, strategy, table_name="
 
         self.create_table(table_name, keyspace_name=keyspace_name)
 
+    def download_file_from_backup_repo(self, local, bucket, cluster_id):
+        self.log.info('Will download files for each db machine')
+        cmd = f'''sudo yum install -y python-pip
+                  sudo pip install awscli
+                  sudo pip install boto3
+                  mkdir -p {local}'''
+        for node in self.db_cluster.nodes:
+            node.remoter.run(f'bash -cxe "{cmd}"')
+            nodetool_info = self.db_cluster.get_nodetool_info(node)
+            node_id = nodetool_info['ID']
+            region_name = nodetool_info['Data Center']
+            source_path = f's3://{bucket}/backup/sst/cluster/{cluster_id}/dc/{region_name}/node/{node_id}/keyspace/'
+            download_cmd = f'aws s3 cp {source_path} {local} --recursive'
```

Comment:
since the backup process is taking place on the nodes themselves via the agent,
this is a temporary credentials that is being used, and this download of the backup for restore is also temporary sine manager doesn't support it yet

Reply:
this is using a instance profile, created by #1514